plugins/chat/app/services/chat/auto_leave_channels.rb

```diff
@@ -17,6 +17,8 @@ class AutoLeaveChannels
       attribute :group_id, :integer
       attribute :channel_id, :integer
       attribute :category_id, :integer
+
+      validates :event, presence: true
```

Comment:
Decided to make the `event` param required so we can add it in the comment in the SQL queries for better debugging